### PR TITLE
Create seed file for populating the database

### PR DESCRIPTION
**Pull request by nikolatriki**
Notion task: https://www.notion.so/Create-seed-file-with-Faker-571cd522226147f6bebef18eaec62d67

### CODE REVIEW

**Review by violeta-p — CHANGES_REQUESTED** [review]
There's no need to loop again through giveaways. In the first loop, once the giveaway is created you can create two comments for that giveaway. We can pair-up if needed.

**Comment by nikolatriki** [comment]
> There's no need to loop again through giveaways. In the first loop, once the giveaway is created you can create two comments for that giveaway. We can pair-up if needed.

No problem, I did only chain the methods and it worked! I assume that it's OK, and it's correct way?

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,16 @@
...
+
+  30.times do
+    Giveaway.create(
+    title: Faker::Commerce.product_name,
+    description: Faker::Lorem.paragraphs,
+    location: Faker::Address.city
+    ).comments.create(
+          commenter: Faker::Movies::Lebowski.character,
+          body: Faker::Movies::Lebowski.quote
+        )
```
Comment (violeta-p) [inline db/seeds.rb:17]: I believe it's better to store creating giveaways in one variable, and then create two comments per giveaway.

Something like:
```
giveaway =  Giveaway.create(
    title: Faker::Commerce.product_name,
    description: Faker::Lorem.paragraphs,
    location: Faker::Address.city
    )

2.times do |c|
  giveaway..comments.create(
          commenter: Faker::Movies::Lebowski.character,
          body: Faker::Movies::Lebowski.quote
        )
end
```

Like this, you have separate logic and more control over it.

Comment (nikolatriki) [reply db/seeds.rb:17]: @violeta-p  How to do that with 10 giveaways with 2 comments each?

**Review by violeta-p — APPROVED** [review]

**Pull request merged**